### ISSUE TITLE
combine swapin + swapout into one check

### ISSUE THREAD

**Issue by BorisTrnka** [NagiosEnterprises/ncpa#846]
Hello,
  with a reference to https://github.com/NagiosEnterprises/ncpa/issues/674 would it be possible to combine these to metrics into one check-result?
  
  I mean, next to having "api/memory/swap/swapped_out" and "api/memory/swap/swapped_in" to have also "api/memory/swap/swapped" which would return both values in the result (similarly as "api/memory/swap" combines the "used", "free" and "total").
  
  Thanks.

**Comment by jomann09** [comment]
Yeah I think that this is probably a good idea, we can definitely mark it as an enhancement, might be able to do this in a version after 3.
